docs/control_tuple.md

```diff
@@ -0,0 +1,207 @@
+# Custom Control Tuple (User Defined Control Tuple) Support
```

Comment:
Since the headings can be used to link to the docs, can we use a simple heading such as "User Defined Control Tuples"